apps/project/schema.py

```diff
@@ -581,6 +618,20 @@ class Meta:
         filterset_class = PublicProjectByRegionGqlFileterSet
 
 
+class ProjectSummaryStatEntryActivityType(graphene.ObjectType):
+    project_id = graphene.ID()
+    count = graphene.Int()
+    date = graphene.Date()
+
+
+class ProjectSummaryStatType(graphene.ObjectType):
```

Comment:
Let's also rename this to UserProjectSummaryStatType and UserProjectSummaryStatEntryActivityType

Reply:
```diff
- project_stat_summary = graphene.Field(UserProjectSummaryStatType, required=True)
+ user_project_stat_summary = graphene.Field(UserProjectSummaryStatType, required=True)
```